Description: Fix paths in tools/rehash
 Incorporates and hence supersedes some parts of
 0006-Fix-paths-on-Debian-in-tools-rehash.patch. The latter
 actually changes both tools/rehash and perl/imap/IMAP/Shell.pm,
 which is modified in a different patch in our series.
Author: Sven Mueller <debian@incase.de>
Forwarded: no
Last-Update: 2022-05-05

---
 tools/rehash | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/rehash b/tools/rehash
index bf563336d..962e9dce8 100755
--- a/tools/rehash
+++ b/tools/rehash
@@ -66,10 +66,10 @@ my %ishash = map { $_ => 1 } (@bdirs, @fdirs);
 
 # evil globals
 my @configs = ($imapdconf);
-my $confdir = "/var/imap";
+my $confdir = "/var/lib/cyrus";
 my @parts;
 my $yn = "y";
-my $sievedir = "/usr/sieve";
+my $sievedir = "/var/spool/sieve";
 my $nosievedir = 0;
 my $hashimapspool = 0;
 my $fulldirhash = 0;
